cmd/coder/urls.go

```diff
@@ -35,47 +36,52 @@ var urlAccessLevel = map[string]string{
 
 func portIsValid(port string) bool {
 	p, err := strconv.ParseUint(port, 10, 16)
-	if p < 1 {
+	if p < 1 || p > 65535 {
 		// port 0 means 'any free port', which we don't support
 		err = strconv.ErrRange
 	}
 	if err != nil {
-		fmt.Println("Invalid port")
+		flog.Error("Invalid port")
 	}
 	return err == nil
 }
 
 func accessLevelIsValid(level string) bool {
 	_, ok := urlAccessLevel[level]
 	if !ok {
-		fmt.Println("Invalid access level")
+		flog.Error("Invalid access level")
 	}
 	return ok
 }
 
 type createSubCmd struct {
-	access string
+	access  string
+	urlname string
 }
 
 func (sub *createSubCmd) RegisterFlags(fl *pflag.FlagSet) {
 	fl.StringVarP(&sub.access, "access", "a", "private", "[private | org | authed | public] set devurl access")
+	fl.StringVarP(&sub.urlname, "name", "n", "", "devurl name")
 }
 
 func (sub createSubCmd) Spec() cli.CommandSpec {
 	return cli.CommandSpec{
 		Name:  "create",
```

Comment:
I wonder if we should define a synonym `update` for user-friendliness. `create` works for both insert and update of devurls.